ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -153,3 +153,32 @@ def test_torch_matrix_power(
         input=x,
         n=n,
     )
+
+@handle_frontend_test(
+    fn_tree="torch.linalg.eigh",
+    dtype_and_x=_get_dtype_and_square_matrix(),
+)
+def test_torch_eigh(
+    *,
+    dtype_and_x,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        all_aliases=["eigh"],
```

Comment:
I can't seem to find `torch.eigh` other than `torch.linalg.eigh`, this is supposed to be only used in case of multiple name for the same function. Is this the case here?